starky/src/get_challenges.rs

```diff
@@ -153,13 +136,8 @@ where
 
     challenger.observe_cap(trace_cap);
 
-    let permutation_challenge_sets = permutation_zs_cap.map(|permutation_zs_cap| {
-        let tmp = get_n_permutation_challenge_sets_target(
-            builder,
-            &mut challenger,
-            num_challenges,
-            stark.permutation_batch_size(),
-        );
+    let lookup_challenge_set = auxiliary_polys_cap.map(|permutation_zs_cap| {
+        let tmp = get_grand_product_challenge_set_target(builder, &mut challenger, num_challenges);
         challenger.observe_cap(permutation_zs_cap);
         tmp
```

Comment:
This is repeating code. Can we do better?

Reply:
What do you mean this is repeating code?

Reply:
My bad, it looked a lot like the code starting on line #40, but it's actually not the same.